test/components/empty_state_component_test.rb

```diff
@@ -0,0 +1,14 @@
+require "test_helper"
+
+class EmptyStateComponentTest < ViewComponent::TestCase
+  test "Renders the title, body and icon" do
+    title = "No notifications"
+    body = "Nothing to see"
+
+    render_inline(EmptyStateComponent.new(title: title, body: body, icon: "icons/outline/bell.svg"))
+
+    assert_selector("h3", text: title)
+    assert_selector("p", text: body)
+    assert_selector("svg")
```

Comment:
If you use an SVG from `icons/brands/` they have the `title` attribute set. And you can test them like `test/components/social_link_component_test.rb`.

Reply:
Oh!! Neat